feature/interface/singleton/ate_tests/singleton_test/singleton_test.go

```diff
@@ -417,17 +430,24 @@ func (tc *testCase) testFlow(t *testing.T, packetSize uint16, ipHeader ondatra.H
 	} else if avg := octets / outPkts; avg > uint64(tc.mtu) {
 		t.Errorf("Flow source packet size average got %d, want <= %d (MTU)", avg, tc.mtu)
 	}
-	if p1InDiff.unicast < outPkts {
-		t.Errorf("DUT received too few source packets: got %d, want >= %d", p1InDiff.unicast, outPkts)
-	}
 
 	if inPkts == 0 {
```

Comment:
Where is the end curly brace for
if inPkts == 0 {
since you deleted line 428, 429, and a new if block starts from 439?

This will cause syntax error, right? can this be compiled? Does GH do such kind of sanity check?

Reply:
Please check this block of code, it is passing and i  will attach passed log.

if inPkts == 0 {
		// The PacketLargerThanMTU cases do not expect to receive packets,
		// so this is not an error. Ethernet counters will be incremented
		// in case of Packets which larger than MTU
		t.Log("Flow did not receive any packet")
		if p1InEthDiff.unicast < outPkts {
			t.Errorf("DUT received too few source packets: got %d, want >= %d", p1InEthDiff.unicast, outPkts)
		}
	} else {
		if p1InDiff.unicast < outPkts {
			t.Errorf("DUT received too few source packets: got %d, want >= %d", p1InDiff.unicast, outPkts)
		}
		if avg := octets / inPkts; avg > uint64(tc.mtu) {
			t.Errorf("Flow destination packet size average got %d, want <= %d (MTU)", avg, tc.mtu)
		}
	}